pump/storage/storage_test.go

```diff
@@ -298,6 +299,23 @@ func (as *AppendSuit) TestDoGCTS(c *check.C) {
 	}
 }
 
+func (as AppendSuit) TestWaitL0FilesCompacted(c *check.C) {
```

Comment:
this can't test `l0Num >= l0Trigger` -> l0Num < l0Trigger, it's a little confusing about the name  `TestWaitL0FilesCompacted`

Reply:
https://github.com/pingcap/tidb-binlog/pull/736/commits/2b07c08baf67a47d19a8d2f6d2e1f19fbfb6041c

what about this kind of implementation which use failpoint?

Reply:
https://github.com/pingcap/tidb-binlog/pull/736/commits/2b07c08baf67a47d19a8d2f6d2e1f19fbfb6041c

what about this kind of implementation which use failpoint?